deps/3rdparty/libdispatch/build/libdispatch.vcxproj

```diff
@@ -249,19 +248,14 @@
     <ClangCompile Include="..\src\shims\mach.c" />
   </ItemGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
-
   <PropertyGroup>
     <PrebuiltPath Condition="'$(PrebuiltPath)' == ''">$(WINOBJC_SDK_ROOT)\deps\prebuilt\Universal Windows\$(PlatformTarget)\$(Configuration)\\</PrebuiltPath>
   </PropertyGroup>
-  <Target Name="CopyContent" AfterTargets="Build">
-    <ItemGroup>
-      <LibraryOutput Include="$(OutDir)\*.lib" />
-      <LibraryOutput Include="$(OutDir)\*.dll" />
-      <LibraryOutput Include="$(OutDir)\*.pdb" />
-      <Headers Include="..\dispatch\*.h" />
-      <Headers Include="..\src\*.h" />
-    </ItemGroup>
-    <Copy DestinationFolder="$(PrebuiltPath)" SkipUnchangedFiles="True" SourceFiles="@(LibraryOutput)" />
-    <Copy DestinationFolder="$(WINOBJC_SDK_ROOT)\deps\prebuilt\include\dispatch\\" SkipUnchangedFiles="True" SourceFiles="@(Headers)" />
-  </Target>
+    <Target Name="AddBuiltOutput" BeforeTargets="GetPackageContents">
+        <ItemGroup>
+            <PackageFile Include="@(BuiltProjectOutputGroupOutput -> '%(FinalOutputPath)')">
```

Comment:
marginally surprised that you can use a -> operator here

Reply:
This is doing an ItemGroup transform to make a new list that is the %(FinalOutputPath) property of all the items in BuiltProjectOutputGroupOutput.

That said, what is going on here? Why are you adding packaging information here? Are we going to start building this as part of tools.sln instead of as a prebuilt?

---
In reply to: [141208433](https://github.com/Microsoft/WinObjC/pull/2799#discussion_r141208433) [](ancestors = 141208433)

Reply:
I see looks like just dispatch would be a package. What is the value of breaking this out? Is this so we could ship different versions of dispatch / ship them on a different cadence?

---
In reply to: [141459971](https://github.com/Microsoft/WinObjC/pull/2799#discussion_r141459971) [](ancestors = 141459971,141208433)

Reply:
also though I'd expect that you'd remove the checked in dispatch binaries if thats what you are intending to do.

---
In reply to: [141460824](https://github.com/Microsoft/WinObjC/pull/2799#discussion_r141460824) [](ancestors = 141460824,141459971,141208433)